bazel/repos.bzl

```diff
@@ -85,6 +85,33 @@ def repos(external = True, repo_mapping = {}):
         repo_mapping = repo_mapping,
     )
 
+    # Copied and then modified to use 'commit' and 'shallow_since'
+    # rather than tracking the 'master' branch from:
+    # https://github.com/fmtlib/fmt/tree/master/support/bazel
+    maybe(
+        git_repository,
+        name = "com_github_fmtlib_fmt",
+        commit = "b6f4ceaed0a0a24ccf575fab6c56dd50ccf6f1a9",
+        shallow_since = "1641508515 -0800",
+        remote = "https://github.com/fmtlib/fmt",
+        patch_cmds = [
```

Comment:
Given these patch commands, would it be worth adding something like a bazel rules repo for fmtlib?

Reply:
I think we should follow build pattern which is described [`here`](https://github.com/fmtlib/fmt/tree/master/support/bazel).